chef-acceptance.gemspec

```diff
@@ -21,7 +21,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rspec'
   s.add_development_dependency 'chef'
   s.add_development_dependency 'gem-release'
-  s.add_development_dependency 'chefstyle'
```

Comment:
are we removing this to ensure we are always pulling from chefstyle master?  :+1: